lib/jsdom/level2/html.js

```diff
@@ -288,10 +288,15 @@ inheritFrom(core.Document, core.HTMLDocument, {
     })(null, true);
   },
 
-  // document.write is defined in browser/index.js.
+  // document.write is defined in level1/core.js.
```

Comment:
Want to move this up to level1/core.js, to help with the eventual collapsing goal?